website/blog/releases/3.0/index.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: Announcing Docusaurus 3.0
+authors:
+  - slorber
+tags: [release]
+image: ./img/social-card.png
+date: 2023-10-25
+unlisted: true
+---
+
+Today we are happy to **announce Docusaurus 3.0**! 🥳
+
+At [**Meta Open Source**](https://opensource.fb.com/), we believe Docusaurus will help you build the **best documentation websites** with **minimal effort**, letting you **focus on what really matters**: writing the content.
+
+This is a new **major version** of Docusaurus, coming with **new exciting features** and upgraded dependencies.
+
+We respect [**Semantic Versioning**](https://semver.org/) and this release also includes **breaking changes** that we documented thoroughly in the [**v3 upgrade guide**](/docs/3.0.0-rc.0/migration/v3). Breaking changes can be annoying, but are necessary to set the ground for a **new wave of Docusaurus features** that we plan to implement later.
+
+![v3.0 social-card image](./img/social-card.png)
+
+{/* truncate */}
+
+We initially planned to release more frequent major versions, but this one has taken longer than expected. **Upgrading to [MDX v2](https://mdxjs.com/blog/v2/) is the main challenge** to the adoption of this new version. We went the extra mile to make this upgrade as easy as possible, notably by adding [compatibility options for MDX v1](/docs/3.0.0-rc.0/api/docusaurus-config#markdown).
```

Comment:
We initially planned to release more frequent major versions, but Docusaurus v3 has taken longer than expected.

Reply:
```suggestion
We initially planned to release more frequent major versions, but Docusaurus v3 has taken longer than expected. **Upgrading to [MDX v2](https://mdxjs.com/blog/v2/) is the main challenge** to the adoption of this new version. We went the extra mile to make this upgrade as easy as possible, notably by adding [compatibility options for MDX v1](/docs/3.0.0-rc.0/api/docusaurus-config#markdown).
```